POEApi.Model/BreachStone.cs

```diff
@@ -2,11 +2,11 @@
 
 namespace POEApi.Model
 {
-    public class BreachBlessing : StackableItem, IBreachCurrency
+    public class BreachStone : StackableItem, IBreachCurrency
```

Comment:
According to the [wiki](https://pathofexile.gamepedia.com/Breachstone), this should be `Breachstone` (note the capitalization).

Reply:
Done